app/services/claims/import_schools.rb

```diff
@@ -0,0 +1,19 @@
+require "csv"
+
+class Claims::ImportSchools
+  include ServicePattern
+
+  def initialize(csv_file_path:)
+    @csv_file_path = csv_file_path
+  end
+
+  attr_reader :csv_file_path
```

Comment:
Could this be a private `attr_reader`?

Reply:
As a general rule we should always default to using `private` unless we need something (an attribute, method, etc) to be publicly accessible, this gives us confidence that we can make changes to the logic without breaking things outside the scope of the class.

Happy for others to weigh in here though!